packages/api/src/auth/strategies/github.client.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy } from 'passport-github2';
+import { VerifiedCallback } from 'passport-jwt';
+
+import { ConfigService } from '../../shared/config.service';
+import { GithubDTO, GithubUserData } from '../auth.model';
+
+@Injectable()
+export class GithubClient extends PassportStrategy(Strategy) {
+  constructor(configService: ConfigService) {
+    super(configService.githubClientConfig);
+  }
+
+  async validate(_token: string, _refreshToken: string | undefined, profile: GithubDTO, done: VerifiedCallback) {
```

Comment:
It might be shorter to write `_refreshToken?: string` instead of `_refreshToken: string | undefined`

Reply:
There is a difference between those two, as I understand it well.
With the question mark, you can ommit the parameter, with string | undefined you need to explicity tell that this param is undefined.